utils.js

```diff
@@ -749,7 +749,7 @@ const getPyMetadata = async function (pkgList, fetchIndirectDeps) {
       }
     }
   }
-  if (indirectDeps.length && fetchIndirectDeps) {
+  if (fetchIndirectDeps) {
```

Comment:
Forgot the and condition. I don't think this would fix the underlying issue. Let's troubleshoot this in a different manner by adding more console.log statements.